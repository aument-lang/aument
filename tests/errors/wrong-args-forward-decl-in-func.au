func a() {
    b(1,2,3);
}

func b() {}