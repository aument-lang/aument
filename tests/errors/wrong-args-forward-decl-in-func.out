parser error(4) in -: wrong number of arguments (expected 0, got 3)
2 |     b(1,2,3);
        ^
